### DISCUSSION TITLE
new plugin: mark-news

### DISCUSSION THREAD

**Discussion by rejetto** [rejetto/hfs#584] [Show and tell]
i just made a plugin, it's called mark-news.
Check it out
https://github.com/rejetto/mark-news

**Comment by PH5HP** [comment]
Hi,

Nice one... :-)  I sent you some links about chats in Discord.

Kind regards,
Henk - ph5hp.nl - https://fileserver.ph5hp.nl

**Comment by PH5HP** [comment]
Hi Rejetto,

The new plugin is working as supposed to. I was trying to add a 'NEW' image, but no luck yet. Is this possible with this plugin? I get the image on the screen, but not behind the line. Maybe I am doing something wrong? 

Kind regards,
Henk - ph5hp.nl - https://fileserver.ph5hp.nl/

**Reply by rejetto** [reply]
what you mean with "behind the line"? show what you are doing

**Reply by PH5HP** [reply]
Hi Rejetto,

It;s simple :-)

I want to replace *NEW* with an image.

Kind regards,
Henk - ph5hp.nl - https://fileserver.ph5hp.nl/
![image](https://github.com/rejetto/hfs/assets/140949730/917facf5-20a0-478f-a54f-865ce02f67c8)
![image](https://github.com/rejetto/hfs/assets/140949730/6ee01fe0-e558-48b4-bbf8-a8225e2b854e)

**Comment by rejetto** [comment]
your problem is just about html/css, nothing to do with the plugin.
try `<img src="/images/New.png" style="width:30px">`

**Reply by PH5HP** [reply]
Hi,

TNX. Almost there.

Kind regards,
Henk - ph5hp.nl - https://fileserver.ph5hp.nl/

![image](https://github.com/rejetto/hfs/assets/140949730/cd1d4aac-debe-4547-a21f-070abce4d23a)

**Reply by rejetto** [reply]
your problem was caused by some custom css (not HFS, yours or some plugin's) that is giving width:100% for any img

**Reply by PH5HP** [reply]
Hi,

Yes, you are right. I corrected this in the CSS section and now it looks Ok. I even made a nice NEW image.

Kind regards,
Henk - ph5hp.nl - https://fileserver.ph5hp.nl/

![image](https://github.com/rejetto/hfs/assets/140949730/0e85c3cd-3a71-4524-95b6-4edcc293fa58)

**Comment by GeorgeGeorgeGe** [comment]
it worked perfect!
Thanks a lot.
Best Regards.
